astropy/coordinates/builtin_frames/intvis.py

```diff
@@ -0,0 +1,113 @@
+
+from astropy.coordinates.attributes import (CoordinateAttribute, EarthLocationAttribute,
+                                            TimeAttribute, QuantityAttribute)
+
+from astropy.coordinates.earth import EarthLocation
+
+from astropy.coordinates.transformations import FunctionTransform
+from astropy.coordinates.representation import (SphericalRepresentation,
+                              UnitSphericalRepresentation,CartesianRepresentation)
+
+from . import (ITRS, ICRS)
+               
+class InterferometricVisibility(BaseCoordinateFrame):
+    """
+    A coordinate or frame in the interferometric visibilities (a.k.a. "uvw"),
+    as is common in radio astronomy.
+    This system are the local East, North, and Up unit vectors projected
+    into the plane of the sky, whose origin is fixed to a direction ``phase``.
+    These coordinates are used to measure the projected antenna position
+    that can be differenced against others for a baseline measurement.
+
+    This frame has the following frame attributes, which are necessary for
+    transforming from InterferometricVisibility to some other system:
+    * ``obstime``
+        The time at which the observation is taken.  Used for determining the
+        position and orientation of the Earth.
+    * ``location``
+        The location of the center of the radio interferometer on the Earth,
+        typically the centroid of all antenna locations, though it can be
+        fixed to any particular antenna.  This can be specified either as an
+        `~astropy.coordinates.EarthLocation` object or as anything that can be
+        transformed to an `~astropy.coordinates.ITRS` frame.
+    * ``phase``
+        The phase tracking center of the frame.  This can be specified either
+        as an (ra,dec) `~astropy.units.Quantity` or as anything that can be
+        transformed to an `~astropy.coordinates.ICRS` frame.
+
+    Parameters
+    ----------
+    representation : `BaseRepresentation` or None
+        A representation object or None to have no data (or use the other
+        keywords)
+    u : :class:`~astropy.units.Quantity`, optional, must be keyword
+        The u coordinate for this object (``v`` and ``w`` must also be given
+        and ``representation`` must be None).
+    v : :class:`~astropy.units.Quantity`, optional, must be keyword
+        The v coordinate for this object (``u`` and ``w`` must also be given
+        and ``representation`` must be None).
+    w : :class:`~astropy.units.Quantity`, optional, must be keyword
+        The w coordinate for this object (``u`` and ``v`` must also be given
+        and ``representation`` must be None).
+
+    Notes
+    -----
+    It enables simple transformation between various other frames and the
+    observers frame during the radio synthesis.
+    It enables easily calculating elevation of a field with e.g.
+    #>>> coord = astropy.coordinates.SkyCoord(ra=..., dec=...,
```

Comment:
Perhaps modify this example to be doctest-able? And remove the leading `#`?

Reply:
I'll push a commit that modifies this, but I'm not clear on how to set doctests here. I'll make sure the test in `test_frames.py` is good.